vterm.el

```diff
@@ -410,6 +410,12 @@ The units are seconds.")
            :sentinel (when (or vterm-exit-functions
                                vterm-kill-buffer-on-exit)
                        #'vterm--sentinel))))
+
+  ;; Change major-mode is not allowed
+  (add-hook 'change-major-mode-hook
+            #'(lambda () (interactive)
+                (user-error "You cannot change major mode in vterm buffers")) -90 t)
```

Comment:
-90? I think you want `nil` for append.

Reply:
I want to prepend. The error has to be thrown immediately, or Emacs will crash.

Reply:
Any non-nil value (which -90 is) will append. Only `nil` will prepend.

Reply:
Thanks for catching that. I experimented with `nil`, but it seemed that it didn't work. Now it's fixed.